src/Microsoft.Health.Dicom.Core/Features/Validation/ValidationErrorCode.cs

```diff
@@ -0,0 +1,69 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Validation
+{
+    /// <summary>
+    /// Validation Error Code.
+    /// Error Code is a short number (from 0 to 32767), and organized in this way:
+    /// 0 => No error.
+    /// [1,100) => General error for all VR.
+    /// [100,200) => Error for PN.
+    /// [200,300) => Error for DA.
+    /// Each VR could have up to 100 error code.
+    /// </summary>
+    public enum ValidationErrorCode
+    {
+        /// <summary>
+        /// No error.
+        /// </summary>
+        None = 0,
```

Comment:
Is this even valid? ErrorCode enum and none

Reply:
it for code analysis CA1008
![image](https://user-images.githubusercontent.com/67345894/128730922-5e4b09e1-8a6c-43e3-a63a-19e8f0490cc4.png)